peps/pep-0815.rst

```diff
@@ -9,6 +9,7 @@ Type: Standards Track
 Topic: Packaging
 Created: 04-Dec-2025
 Post-History: `09-Jun-2025 <https://discuss.python.org/t/94968>`__,
+              `08-Dec-2025 <https://discuss.python.org/t/105232>`__,
```

Comment:
Please can you also add a Discussions-To pointing to this? 

It's used for whatever is the current thread.

Reply:
And the linter is complaining about the order:

peps/pep-0815.rst:1:  Headers must be in PEP 12 order. Correct order: PEP, Title, Author, Sponsor, PEP-Delegate, Discussions-To, Status, Type, Topic, Created, Post-History
check-peps failed: 1 error